go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/ec2 v1.16.0
 	github.com/aws/aws-sdk-go-v2/service/sts v1.7.0
 	github.com/beevik/etree v1.1.0
-	github.com/coreos/go-oidc v0.0.4
+	github.com/coreos/go-oidc v2.1.0+incompatible
```

Comment:
This is a big jump - should we do some testing to make sure nothing else breaks? 

@r0mant, thoughts?

Reply:
@tigrato I would probably skip v9 and v8 backports altogether tbh. We can tell folks to upgrade to v10 if they need this.

Reply:
@r0mant got it!